qiskit_ibm_runtime/api/clients/runtime.py

```diff
@@ -131,6 +132,7 @@ def program_run(
             image: The runtime image to use.
             hgp: Hub/group/project to use.
             log_level: Log level to use.
+            session_id: Job ID of the first job in a runtime session.
```

Comment:
This is not a public class so users wouldn't know :) Would we as developers understand if I just call it `ID of the runtime session` ?

Reply:
Good point that it's not a public class. Then my only very minor remark: `Job ID of the first job sounds redundant` (I had to read it twice because I got confused initially). `ID of the first job` might be clearer unless there will be multiple ids associated with a job.

Reply:
```suggestion
            session_id: ID of the first job in a runtime session.
```